src/molecules/product-list-item/ProductListItem.stories.js

```diff
@@ -0,0 +1,131 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import AProductListItem from './ProductListItem.vue'
+
+const info = 'Product List Item component'
+
+// @vue/component
+const defaultData = {
+  components: { AProductListItem },
+  data () {
+    return {
+      id: 'prod1',
+      qty: 2,
+      options: [
+        {
+          'id': 'size_id_1',
+          'label': 'Size',
+          'value': 'XXL'
+        },
+        {
+          'id': 'color_id_1',
+          'label': 'Color',
+          'value': 'Red'
+        }
+      ]
+    }
+  },
+  methods: {
+    removeMethod: action('Remove'),
+    changeMethod: action('Change')
+  }
+}
+
```

Comment:
Can we add some knobs?

Reply:
Sure, added for product: name, options, image url and price.